dev-cpp/valijson/valijson-1.0.3.ebuild

```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="Header-only C++ library for JSON Schema validation"
+HOMEPAGE="https://github.com/tristanpenman/valijson"
+SRC_URI="https://github.com/tristanpenman/valijson/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD-2 Boost-1.0"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~x86"
+RESTRICT="test"
```

Comment:
We'd like to have a comment why tests are restricted. They don't exist, they don't work (with upstream issue link)

Reply:
I just forgot to remove the RESTRICT - it's a leftover from https://github.com/tristanpenman/valijson/issues/164
Then the packaging changed and running the tests became impossible, see [here](https://gitweb.gentoo.org/repo/gentoo.git/commit/dev-cpp/valijson?id=44f039de30123a9472590cc6d2c15aa520a1d82b).
Feel free to remove it or I can make a note to do it on the next bump.